doc/modules/tree.rst

```diff
@@ -572,6 +572,32 @@ Mean Absolute Error:
 
 Note that it fits much slower than the MSE criterion.
 
+.. _tree_missing_value_support:
+
+Missing Values Support
+======================
+
+:class:`~tree.DecisionTreeClassifier` and :class:`~tree.DecisionTreeRegressor`
+have built-in support for missing values when `splitter='best'` and criterion is
+`'gini'`, `'entropy`', or `'log_loss'`, for classification' or
+`squared_error'`, `'friedman_mse'`, or `'poisson'` for regression.
+
+For each potential threshold, the splitter will evaluate the split with all the
+missing values going to the left child or the right child. When predicting, the
+samples with missing values are assigned according::
```

Comment:
```suggestion
samples with missing values are assigned accordingly::
```
I think. Or was there more to come a la "... according to some more text here::". I can't think of a short description and I think it is clear as it is. But maybe having an explicit sentence that says "we use the direction that was found to be best during training", but it also feels like a message from the "Department of the Obvious" :-/